### PR TITLE
add landingpage component

### PR DESCRIPTION
**Pull request by wahyuhjr**

### CODE REVIEW

**Review by wahyuhjr — COMMENTED** [review]
add component

File: src/app/components/Card/index.jsx
Comment (indrazm) [inline src/app/components/Card/index.jsx:1]: Do not use  use <Image>

**Review by indrazm — CHANGES_REQUESTED** [review]
Please change

File: src/components/Button/index.jsx
Comment (indrazm) [inline src/components/Button/index.jsx:1]: Don't need `<>`

**Review by indrazm — APPROVED** [review]

**Review by ekmigasari — APPROVED** [review]

**Pull request merged**